jobs.py

```diff
@@ -24,6 +24,7 @@
             "tests/misc",
             "tests/migration",
             "tests/snapshot",
+            "tests/xapi",
             "tests/xapi-plugins",
```

Comment:
Is it related to this PR?

Reply:
Yes. Every new test must belong to a job if we want them to be tested regularly.